newa/cli.py

```diff
@@ -183,191 +195,221 @@ def cmd_jira(ctx: CLIContext, issue_config: str, recreate: bool) -> None:
         raise Exception('Jira URL is not configured!')
 
     for artifact_job in ctx.load_artifact_jobs('event-'):
+        # when issue_config is defined, --issue and --job-recipe are ignored
+        # as it will be set depending on the --issue-config content
+        if issue_config:
```

Comment:
FYI this condition is new but the block inside hasn't been changed except the indentation due to block inclusion under the condition itself.